### ISSUE TITLE
Unable to animate transforms

### ISSUE THREAD

**Issue by holmesal** [necolas/react-native-web#129]
I'm attempting to animate a transform, but it appears to just retain the initial value.

Here's the component:
```javascript
import React, {
	Animated,
	Component,
	Image,
	PropTypes,
	StyleSheet,
	Text,
	View
} from 'react-native';

export default class AnimationTest extends Component {

	state = {
		position: new Animated.Value(0)
	};

	componentDidMount() {
		setTimeout(() => {
			Animated.spring(this.state.position, {
				toValue: 300
			}).start()
		}, 2000);
	}


	render() {
		return (
			<Animated.View style={[styles.wrapper, {
								opacity: this.state.position,	// this works
								transform: [{translateX: this.state.position	// this doesn't work
								}]}]}>
				<Text>I am the AnimationTest component!</Text>
			</Animated.View>
		);
	}
}

let styles = StyleSheet.create({
	wrapper: {
		width: 200,
		height: 200,
		backgroundColor: 'red'
	}
});

```

And the result:
![image](https://cloud.githubusercontent.com/assets/1147390/14732354/d8912d88-080e-11e6-88b3-7df63abdaa96.png)

**Comment by holmesal** [comment]
Just realized that I could animate `left`, which works:
```javascript
render() {
		return (
			<Animated.View style={[styles.wrapper, {
								opacity: this.state.position,	// this works
								left: this.state.position,	// this works
								transform: [{translateX: this.state.position	// this doesn't work
								}]}]}>
				<Text>I am the AnimationTest component!</Text>
			</Animated.View>
		);
	}
```

It would be cool if this worked with `translateX`, though - more native-y 😁

**Comment by bmcmahen** [comment]
I'm running into this issue too. `left` does work, but `translateX` doesn't.

**Comment by necolas** [comment]
This was happening because the transform value was a number and ended up creating inline styles without the `px` unit. It should be fixed in 0.0.31

**Comment by ezsper** [comment]
I endup with the same bug in the 0.0.28, the transform style is not being animated, only rooted styles like opacity,
![notanimatingtransform](https://cloud.githubusercontent.com/assets/798804/17066119/f53aed0c-501b-11e6-97c3-d47162237af6.gif)
https://rnplay.org/apps/8VhSjw
https://github.com/ezsper/react-native-web-playground/blob/master/index.web.js
